_i18n/en.yml

```diff
@@ -42,47 +43,44 @@ explore:
-  question_fifteen: 15. How was the project born?
-  answer_fifteen: The <a href="/sobre" target="_blank">first phase</a> of the project lasted two months was supported by a <a href="https://www.catarse.me/serenata" target="_blank">crowdfunding campaign</a>. Over 1296 helped us make the project hapen.
-  question_sixteen: 16. How can I contribute to the project?
-  answer_sixteen: You can contribute to the project with code, helping to maintain the team, with ideas,  <a href="https://twitter.com/search?q=%23serenatadeamor" target="_blank">retweets</a> and shares. To understand how to get started with our code base, <a href="https://datasciencebr.com/how-does-one-contribute-to-serenata-de-amor-operation-36e3e7b38207#.een62m7or" target="_blank">check this post</a>. To  directly contribute to the maintenance of the project you can tip us at <a href="https://apoia.se/serenata" target="_blank">apoia.se</a>.
-  question_seventeen: 17. What does the name Serenata de Amor mean?
-  answer_seventeen: 'There three reasons: <br>1 - We are  directly inspired by the famous Toblerone Affair – a famous case in which a politician in Sweden resigned after being caught with a simple Toblerone in the invoice of her corporate card. We want to do this: find corruption at small scale, but in large volume. <br>2 - It seems to be the a nonsense operation name typical of the Brazilian Federal Police, and this is very cool. <br>3 - It literally means love serenade — so this is our love serenade to Brazil.'
-  question_eighteen: 18. Can I use the source code of Serenata de Amor in my city, state or country?
-  answer_eighteen: Sure, this idea is awesome — and also this idea is one of the main purposes of our adoption of open source.
+  question_one: 1. What does the name Serenata de Amor mean?
+  answer_one: 'There three reasons: <br>1 - We are  directly inspired by the famous Toblerone Affair – a famous case in which a politician in Sweden resigned after being caught with a simple Toblerone in the invoice of her corporate card. We want to do this: find corruption at small scale, but in large volume. <br>2 - It seems to be the a nonsense operation name typical of the Brazilian Federal Police, and this is very cool. <br>3 - It literally means love serenade — so this is our love serenade to Brazil.'
+  question_two: 2. Who is responsible for the project?
+  answer_two: 'We are a <a href="https://twitter.com/cuducos/status/819520770716090368" target="_blank"> group </a> of technology enthusiasts. The project started as an initiative of the data scientist <a href="twitter.com/irio" target="_blank">Irio Musskopf</a>, grew and today has taken on a life of its own, with a team of 10 professionals. Every code is open, and now we have hundreds of volunteers: collaborators from all over the world, including journalists and researchers.'
```

Comment:
We are a  group of technology enthusiasts. The project started from the data scientist <a href="twitter.com/irio" target="_blank">Irio Musskopf</a>'s initiative. It has grown considerably and today we are a team of 10 professionals. Every code is open, and we have hundreds of volunteers: collaborators from all over the world, including journalists and researchers.